easybuild/easyconfigs/h/Horovod/Horovod-0.26.1-foss-2021b-CUDA-11.4.1-PyTorch-1.11.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'Horovod'
+version = '0.26.1'
+local_pt_version = '1.11.0'
+local_cuda_suffix = '-CUDA-%(cudaver)s'
+versionsuffix = local_cuda_suffix + '-PyTorch-%s' % local_pt_version
+
+homepage = 'https://github.com/uber/horovod'
+description = "Horovod is a distributed training framework for TensorFlow."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('flatbuffers', '2.0.0'),
+]
+dependencies = [
+    ('Python', '3.9.6'),
+    ('PyYAML', '5.4.1'),
+    ('CUDA', '11.4.1', '', True),
```

Comment:
Please use `SYSTEM` instead of `True` (that's what is tripping up the CI right now)